client/src/icons/AngleRightIcon.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+type Props = { className?: string };
+
+export default ({ className = "" }: Props) => (
```

Comment:
If you are using CRA, you can add `className` to svg-icons just fine. So I would have this module just as svg-module instead, no need for tsx.